src/mainwindow.ui

```diff
@@ -2541,6 +2547,11 @@ p, li { white-space: pre-wrap; }
     <string>Manage dictionaries</string>
    </property>
   </action>
+  <action name="actionasd">
```

Comment:
Do we need this action?

Reply:
I don't think so.
Will remove it as well